platforms/chibios/boards/GENERIC_STM32_G431XB/configs/hal_lld.h

```diff
@@ -0,0 +1,8 @@
+// Copyright 2024 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma 
+
+// TODO: Remove when upstream no longer requires patching
+#include "stm32_registry.h"
+
+#include_next "hal_lld.h"
```

Comment:
```suggestion
#include_next <hal_lld.h>
```
Does not matter much (unlike `#include`, the behavior of `#include_next` does not change), but we apparently have a lint rule for that.